clang/lib/Driver/ToolChains/Arch/LoongArch.cpp

```diff
@@ -14,6 +14,7 @@
 #include "clang/Driver/Options.h"
 #include "llvm/TargetParser/Host.h"
 #include "llvm/TargetParser/LoongArchTargetParser.h"
+#include <string>
```

Comment:
Is this header necessary?

Reply:
It's added automatically by the lsp clangd, I will delete it.